src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -54,6 +54,11 @@ public static boolean isCloudStorageUrl(final String path) {
         return path.startsWith(GCS_PREFIX);
     }
 
+    public static boolean isCloudStorageUrl(final java.nio.file.Path path) {
+        // the initial "" protects us against a null scheme
+        return ("" + path.toUri().getScheme() + "://").equals(GCS_PREFIX);
```

Comment:
Should probably also add a quick unit test that passes in a `Path` with a null scheme to this method.